src/participant/prompts/query.ts

```diff
@@ -3,14 +3,47 @@ import * as vscode from 'vscode';
 import { getHistoryMessages } from './history';
 
 export class QueryPrompt {
-  static getSystemPrompt(): vscode.LanguageModelChatMessage {
-    const prompt = `You are a MongoDB expert.
-Your task is to help the user craft MongoDB queries and aggregation pipelines that perform their task.
-Keep your response concise.
-You should suggest queries that are performant and correct.
-Respond with markdown, suggest code in a Markdown code block that begins with \'\'\'javascript and ends with \`\`\`.
-You can imagine the schema, collection, and database name.
-Respond in MongoDB shell syntax using the \'\'\'javascript code block syntax.`;
+  static getSystemPrompt({
+    databaseName = 'mongodbVSCodeCopilotDB',
+    collectionName = 'results',
```

Comment:
How should we decide on these? Maybe `test` db with `mongoDBVSCodeCopilot` as the collection instead?

Reply:
In playgrounds, we use `mongodbVSCodePlaygroundDB.sales`. I remember Max didn't want to use the `test` database, because it exists by default, and some users might use it extensively. If align these experiences, the `mongodbVSCodeCopilotDB` seems a good match. So maybe then `mongodbVSCodeCopilotDB.test`?